src/Nuget.Dockerfile

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+FROM mcr.microsoft.com/dotnet/aspnet:8.0.3-alpine3.19-amd64@sha256:a531d9d123928514405b9da9ff28a3aa81bd6f7d7d8cfb6207b66c007e7b3075 AS runtimebase
+
+LABEL maintainer="Ed-Fi Alliance, LLC and Contributors <techsupport@ed-fi.org>"
+
+RUN apk --no-cache add gettext=~0 postgresql16-client=~16
+
+FROM runtimebase AS setup
+
+ENV LOG_LEVEL=${LOG_LEVEL}
+#replace with latest
+ARG VERSION=0.0.0-alpha.0.113
+ENV ASPNETCORE_HTTP_PORTS=8080
+ENV OAUTH_TOKEN_ENDPOINT=${OAUTH_TOKEN_ENDPOINT}
```

Comment:
This variable must be set at _runtime_, but it is not required when _building_ an image. In other words, you don't need it here. That's actually true of all the other environment variables below. They're not being used in any way.

Reply:
As discussed in daily standup, we can leave these in place.